src/sql/workbench/contrib/queryplan2/browser/widgets/customZoomWidget.ts

```diff
@@ -53,10 +54,19 @@ export class CustomZoomWidget extends QueryPlanWidgetBase {
 			}
 		};
 
+		const applyButton = new Button(this.container, {
+			title: localize('customZoomApplyButtonTitle', "Zoom (Enter)")
+		});
+		applyButton.setWidth('60px');
```

Comment:
usually setting fixed widths like this can cause some problems with localization.  ideally there is some ability to grow based on content, with some max length.